storage/libradb/src/metrics.rs

```diff
@@ -82,3 +82,49 @@ pub static LIBRA_STORAGE_API_LATENCY_SECONDS: Lazy<HistogramVec> = Lazy::new(||
     )
     .unwrap()
 });
+
+pub static LIBRA_STORAGE_OTHER_TIMERS_SECONDS: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
other timers?

Reply:
Updated the description, hopefully it's easy to understand now.